src/main/app/src/Components/Landscape/Modals/Item/Item.tsx

```diff
@@ -222,6 +222,7 @@ const Item: React.FC<Props> = ({ fullyQualifiedItemIdentifier, small }) => {
     : null;
   const assessmentStatus = assessments ? getItemAssessments(assessments) : [];
   const frameworks: ReactElement | null = item ? getLabelsWithPrefix('framework', item) : null;
+  const networks: ReactElement | null = item ? getLabelsWithPrefix('networks', item) : null;
```

Comment:
As discussed this method is not suitable, since keys and values are similar like in tags. This displays as:

![image](https://user-images.githubusercontent.com/143695/141038848-e4325b38-feb3-4f1b-8d87-c39c8962ccab.png)

Reply:
Yep, I know. But then I thought it makes sense to have it as in frameworks while content is only a placeholder for maybe "network1":  "LAN", "network2: "VPN". But while writing this I realize that network is redundant. So, I will remove it as in tags... ;)

Reply:
For me it was simpler to change the method getLabelsWithPrefix than to do it as with 'tags'.
I added a regexp to getLabelsWithPrefix looking if [key = prefix.value]: value --> only show primary 'content'
hope this is ok for you...

Reply:
Unfortunately no. The regex + 150 chars approach makes it brittle. Please see https://github.com/dedica-team/nivio/pull/760/commits/fbd70551b6a14e582a3c259d825ebff3e3ff32cc as a starting point.

Reply:
Added networks to ItemApiModel.